src/patient-lab-dashboard/table/report-table/report-table.component.tsx

```diff
@@ -114,135 +158,140 @@ const ReportTable = (props: ReportTableProps) => {
-                        const rowIndex = getReportTableRowIndex(
-                          currentPage,
-                          index,
-                        )
-                        return (
-                          <React.Fragment key={row.id}>
-                            <TableExpandRow {...getRowProps({row})}>
-                              {row.cells.map(cell => {
-                                return cell.id.endsWith('file') ? (
-                                  <TableCell key={cell.id}>
-                                    {cell.value?.endsWith('pdf') ? (
-                                      <Link
-                                        href={getReportUrl(rows, row.id)}
-                                        target={'_blank'}
-                                        onClick={() => {
-                                          const auditMessage = getAuditMessageBody(
-                                            patientUuid,
-                                            rows[rowIndex]?.file,
-                                            rows[rowIndex]?.date,
-                                            getPatientIdentifier(
-                                              rows[rowIndex]?.patientId,
-                                            ),
-                                            rows[rowIndex]?.tests,
-                                          )
-                                          postAuditMessage(auditMessage)
-                                        }}
-                                      >
-                                        {cell.value}
-                                      </Link>
-                                    ) : (
-                                      <ImagePreviewComponent
-                                        url={getReportUrl(rows, row.id)}
-                                        fileName={cell.value}
-                                        auditMessage={getAuditMessageBody(
+      <>
+        <h4>Reports table</h4>
+        {(!testResultsError && !labTestResults) ||
+        (!reports && !reportsTableDataError) ? (
+          <Loader />
+        ) : null}
+        <DataTable rows={paginatedReportsTable} headers={reportHeaders}>
+          {({
+            rows: dataTableRows,
+            headers,
+            getHeaderProps,
+            getRowProps,
+            getTableProps,
+            getTableContainerProps,
+          }) => (
+            <TableContainer {...getTableContainerProps()}>
+              <Table
+                {...getTableProps()}
+                className={classes.reportTable}
+                useZebraStyles={true}
+              >
+                <TableHead>
+                  <TableRow>
+                    <TableExpandHeader id="expand" />
+                    {headers.map((header, i) => (
+                      <TableHeader
+                        id={header.header}
+                        key={i}
```

Comment:
please avoid using indices as keys. Attach a string so that it will be unique across the application

Reply:
changes have been made